terraform/cp4ba/main.tf

```diff
@@ -112,31 +112,29 @@ resource "null_resource" "create_DB_Schema" {
   }
 }
 
-  # ------ DB2 -------
+  # ------ CP4BA -------
 module "install_cp4ba"{
-  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/main/modules/cp4ba"
-//  source = "../../../terraform-ibm-cloud-pak/modules/cp4ba"
+  source = "github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//modules/cp4ba"
     depends_on = [
     null_resource.create_DB_Schema
   ]
-
-  KUBECONFIG              = data.ibm_container_cluster_config.cluster_config.config_file_path
-
+  enable_cp4ba           = var.enable_cp4ba
```

Comment:
This can be hardcoded to "true" since it can be assumed we want to provision this module since that is the purpose of this script.

Reply:
Done. I made to a local variable
```
locals {
  enable_cp4ba = true
}
```